### DISCUSSION TITLE
More Layers at Higher Levels

### DISCUSSION THREAD

**Discussion by helloimalastair** [onthegomap/planetiler#537] [Q&A]
Is there something I should do to have more of the layers generate at lower zoom levels? I'm aiming for something akin to the [Streets](https://www.maptiler.com/maps/#style=streets-v2&lang=en&mode=2d&position=0.92/0/-47.2) style, but my map only renders the borders of the country, their names, and the oceans, which I why I assume my tile generation process just isn't outputting the data for higher zoom levels.

I'm using the [OpenMapTiles](https://github.com/openmaptiles/planetiler-openmaptiles) profile.

Target Style:
<img width="944" alt="image" src="https://user-images.githubusercontent.com/37487746/228158842-5e8399b8-00ac-4bdd-b6b2-c2fd30914ff5.png">

My map:
<img width="956" alt="image" src="https://user-images.githubusercontent.com/37487746/228158262-a388fa53-7c21-4781-9b3c-aa522b4576de.png">

**Comment by msbarry** [comment]
The difference you're seeing is that the closed-source maptiler style includes an extra global landcover layer at low zooms that is not included in the open source part. What I did for onthegomap.com is generate image tiles from https://www.naturalearthdata.com/downloads/10m-raster-data/10m-natural-earth-2/ then show them at lower zoom levels. It's also possible to create a comparable low-zoom vector landcover dataset like maptiler has from open data like [ESA Worldcover](https://worldcover2021.esa.int/), although we don't have anything concrete yet.

**Reply by helloimalastair** [reply]
Ok, that makes sense. Is there an issue for adding landcover to `planetiler`?

**Reply by acalcutt** [reply]
There have been several people who I've seen ask maptiler if they would share their process for the landcover, but they haven't wanted to share what they are doing. It would be nice if they would share, but I guess they need to keep some secrets to sell their product :-)

I myself have made a background raster tileset based on JAXA AW3D30 using gdal scripts, which looks like
https://tiles.wifidb.net/data/jaxa_color/#3/41.77/-63.46
(btw, I have this in my google drive here as 'jaxa_color.mbtiles' if you wanted a copy https://drive.google.com/file/d/1wCJvAjSgsb1lQ8HM0WCKDo5jzINTnGFm/view?usp=share_link )
https://github.com/acalcutt/jaxa_AW3D30_to_MBTiles/blob/main/export_color_mbtiles_v2.sh

I have also made one from the ESA data that looks like this
https://tiles.wifidb.net/data/landcover/#2/34.9/11.3
https://www.techidiots.net/notes/mapbox-gl-tileserver-gl/esa-climate-change-initiative-landcover-to-raster-mbtiles

Once you have a background layer, it is easy to add it behind other planetiler layers in a style, like
https://tiles.wifidb.net/styles/WDB_OSM/#3.61/40.37/-99.39
